setup.py

```diff
@@ -46,9 +46,11 @@ def read_version(file_path="version.txt"):
     CUDAExtension,
```

Comment:
I think you might enjoy stack based PR development https://github.com/modularml/stack-pr